### ISSUE TITLE
Can boot using sd card or usb ?

### ISSUE THREAD

**Issue by rhendytay** [devmfc/amlogic-bootscripts-Armbian#2]
Hi can i using this method to boot from usb or sdcard without using emmc ?
Im using HG680P ...

**Comment by devmfc** [comment]
Yes you can, no need to write anything to eMMC, so the existing firmware will not be touched. 

I would recommend USB flash drive over sdcards.

**Comment by rhendytay** [comment]
Hi @devmfc but my stb box did not have a reset button is still can boot without press and hold the reset button ?
Thanks.

**Comment by devmfc** [comment]
You need some way to reset the box / get the box in update mode. 

If there is no reset button, you'll have to find out which way you can reset your box. 
Sometimes it can be done with a remote control button combination or some app in android. 

And of course, you can try to connect a serial adapter to the board and boot the image via the u-boot cli.
